model/span/event.go

```diff
@@ -43,7 +44,7 @@ var (
 
 	spanDocType = "span"
 
-	processorSpanEntry = common.MapStr{"name": "transaction", "event": spanDocType}
```

Comment:
for transactions, this is called `processorTransEntry`, and for errors it `processorEntry`. Let's align them on `processorEntry`?

Reply:
just wanted to make sure you didn't miss this by mistake. It's not critical, so it's fine if you just decided to defer the discussion.

Reply:
I updated now to `processorEntry`.